exercises/wordy/src/test/java/WordProblemSolverTest.java

```diff
@@ -0,0 +1,116 @@
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public final class WordProblemSolverTest {
+
+    /*
+     * See https://github.com/junit-team/junit4/wiki/Rules for information on JUnit Rules in general and
+     * ExpectedExceptions in particular.
+     */
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void testSingleAddition1() {
+        assertEquals(2, new WordProblemSolver().solve("What is 1 plus 1?"));
+    }
+
+    @Ignore
+    @Test
+    public void testSingleAddition2() {
+        assertEquals(55, new WordProblemSolver().solve("What is 53 plus 2?"));
+    }
+
+    @Ignore
+    @Test
+    public void testSingleAdditionWithNegativeNumbers() {
+        assertEquals(-11, new WordProblemSolver().solve("What is -1 plus -10?"));
+    }
+
+    @Ignore
+    @Test
+    public void testSingleAdditionOfLargeNumbers() {
+        assertEquals(45801, new WordProblemSolver().solve("What is 123 plus 45678?"));
+    }
+
+    @Ignore
+    @Test
+    public void testSingleSubtraction() {
+        assertEquals(16, new WordProblemSolver().solve("What is 4 minus -12?"));
+    }
+
+    @Ignore
+    @Test
+    public void testSingleMultiplication() {
+        assertEquals(-75, new WordProblemSolver().solve("What is -3 multiplied by 25?"));
+    }
+
+    @Ignore
+    @Test
+    public void testSingleDivision() {
+        assertEquals(-11, new WordProblemSolver().solve("What is 33 divided by -3?"));
+    }
+
+    @Ignore
+    @Test
+    public void testMultipleAdditions() {
+        assertEquals(3, new WordProblemSolver().solve("What is 1 plus 1 plus 1?"));
+    }
+
+    @Ignore
+    @Test
+    public void testAdditionThenSubtraction() {
+        assertEquals(8, new WordProblemSolver().solve("What is 1 plus 5 minus -2?"));
+    }
+
+    @Ignore
+    @Test
+    public void testMultipleSubtractions() {
+        assertEquals(3, new WordProblemSolver().solve("What is 20 minus 4 minus 13?"));
+    }
+
+    @Ignore
+    @Test
+    public void testSubtractionThenAddition() {
+        assertEquals(14, new WordProblemSolver().solve("What is 17 minus 6 plus 3?"));
+    }
+
+    @Ignore
+    @Test
+    public void testMultipleMultiplications() {
+        assertEquals(-12, new WordProblemSolver().solve("What is 2 multiplied by -2 multiplied by 3?"));
+    }
+
+    @Ignore
+    @Test
+    public void testAdditionThenMultiplication() {
+        assertEquals(-8, new WordProblemSolver().solve("What is -3 plus 7 multiplied by -2?"));
+    }
+
+    @Ignore
+    @Test
+    public void testMultipleDivisions() {
+        assertEquals(2, new WordProblemSolver().solve("What is -12 divided by 2 divided by -3?"));
+    }
+
+    @Ignore
+    @Test
+    public void testExponentiation() {
+        assertEquals(32, new WordProblemSolver().solve("What is 2 raised to the power 5?"));
```

Comment:
What do you think of splitting this out into a separate test class whose name clearly indicates it's bonus tests that can remain ignored?

Also, I'm wondering if we'd like to expand a little on the canonical data set and include more cases for this.  This is inbounds, AFAIK... it's okay to provide additional cases as long as it is sufficiently additive to the experience.  I think the expectation is that we contribute back to x-common.

In general, `wordy` seems to need some love.  I've already submitted a PR to make the x-common descriptive bits... want to address some the weaknesses in the canonical data as well.